proxmox/config_qemu.go

```diff
@@ -849,13 +850,16 @@ storage:xxx
 */
 func (config ConfigQemu) CloneVm(sourceVmr *VmRef, vmr *VmRef, client *Client) (err error) {
```

Comment:
The code you wrote looks good for adding this functionality in to the current state of the code base.

Not happy with how the clone logic works as it shouldn't rely on `ConfigQemu` to this extend and should be changed in the future be that could be a long time.

Reply:
Okay I think I am currently not deep enough into the codebase to do such a big change. Hope that is okay.

Reply:
That's totally fine